packages/docusaurus-theme-redoc/src/theme/Redoc/styles.css

```diff
@@ -126,6 +126,21 @@ html[data-theme='dark']
   background-color: var(--ifm-background-color);
 }
 
+/* Fixes https://github.com/rohit-gohri/redocusaurus/issues/306 */
+html[data-theme='dark'] 
+  .redocusaurus 
+  div[id^='tag']
+  button:has(span):has(.operation-type) {
+  background-color: rgb(27, 32, 40);
```

Comment:
can we use infima css variables (theme of docusaurus classic) rather than hardcoding colors - https://github.com/facebookincubator/infima/blob/main/packages/core/styles/common/variables.pcss?

Reply:
Sure, added now. Tried to keep it consistent with the Server selection dropdown (gray-800) and used emphasis-0 to keep it consistent with the response code drop downs